cypress/e2e/checkout.cy.js

```diff
@@ -1,11 +1,59 @@
+import { faker } from '@faker-js/faker';
+
 /// <reference types='cypress' />
 
-describe('', () => {
-  before(() => {
+const categoryName = 'Laptops';
+const product = 'Sony vaio i7';
+const testMessage = 'Product added';
+const cart = 'Cart';
+const item = 'Sony vaio i7';
+
+const user = {  
+  name: `${faker.name.firstName()} ${faker.name.lastName()}`,  
+  country: 'Canada',  
+  city: 'Ottawa',  
+  creditCard: {    
+    number: faker.finance.creditCardNumber(),    
+    expirationMonth: faker.date.month(),    
+    expirationYear: faker.date.future({ years: 5 }).getFullYear()  
```

Comment:
The method `faker.date.future()` does not accept any arguments. To get a future year, you might want to use `faker.date.future().getFullYear()` without passing an argument.